src/main/resources/rdf/saved_game.ttl

```diff
@@ -0,0 +1,15 @@
+@prefix rdf:  <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix tged: <https://github.com/kdports/TextGameEngine/data/> .
+@prefix tgeo: <https://github.com/kdports/TextGameEngine/ontology/> .
+
+<2>     rdf:type           tgeo:Slide ;
+        tgeo:hasText       "This is a saved game!" ;
+        tgeo:hasXLocation  "0.0" ;
+        tgeo:hasYLocation  "0.0" .
+
+<1>     rdf:type            tgeo:Decision , tgeo:Slide ;
```

Comment:
I assume you didn't hand-write these triples. This is malformed data. The <1> URI should NOT be pointing to both a Decision AND a Slide. Something got confused. I think this has to do with the IdControl being overwritten.

Reply:
I'm going to approve this anyway, but this is a sign of a bug.